website/docs/cloud-docs/vcs/github-app.mdx

```diff
@@ -96,9 +96,8 @@ If you disallow access to a repository that is currently connected to any Terraf
 
 ## Feature Limitations
 
-The Terraform Cloud GitHub App is designed for creating workspaces from the Terraform Cloud UI. It currently does not support:
+The Terraform Cloud GitHub App is designed for creating workspaces and private registry modules from the Terraform Cloud UI. It currently does not support:
```

Comment:
```suggestion
You can use the Terraform Cloud GitHub App to create workspaces and private registry modules from the Terraform Cloud UI. It currently does not support:
```

Suggest changing "is designed for creating" to "You can use...to create" to avoid a passive voice style issue.

Reply:
Thank you for the suggestion, Matthew!